client/wfnews-war/src/main/angular/src/app/components/saved/saved-location-full-details/saved-location-full-details.component.ts

```diff
@@ -0,0 +1,253 @@
+import { ChangeDetectorRef, Component, OnInit, Input } from '@angular/core';
+import { ActivatedRoute, ParamMap, Router } from '@angular/router';
+import { NotificationService } from '@app/services/notification.service';
+import { PointIdService } from '@app/services/point-id.service';
+import { ResourcesRoutes, displayDangerRatingDes, convertToDateYear, getStageOfControlIcon, getStageOfControlLabel, convertToDateTimeTimeZone } from '@app/utils';
+import { SpatialUtilsService } from '@wf1/core-ui';
+import { LocationData } from '../add-saved-location/add-saved-location.component';
+import { PublishedIncidentService } from '@app/services/published-incident-service';
+import { AGOLService } from '@app/services/AGOL-service';
+
+@Component({
+  selector: 'wfnews-saved-location-full-details',
+  templateUrl: './saved-location-full-details.component.html',
+  styleUrls: ['./saved-location-full-details.component.scss']
+})
+export class SavedLocationFullDetailsComponent implements OnInit {
+  @Input() name;
+  location: any;
+  params: ParamMap;
+  public distanceInKm: number = 1;
+  public station: any = [];
+  public hours: any = [];
+  public stationName: string= "";
+  public stationHour: string= "";
+  public fireCentre: string;
+  public dangerRatingLabel: string;
+  public fireBans: any[];
+  public evacOrders: any[] = [];
+  public evacAlerts: any[] = [];
+  public nearbyWildfires: any[];
+
+  displayDangerRatingDes = displayDangerRatingDes
+  convertToDateYear = convertToDateYear
+  getStageOfControlIcon = getStageOfControlIcon
+  getStageOfControlLabel = getStageOfControlLabel
+  convertToDateTimeTimeZone = convertToDateTimeTimeZone
+
+  constructor(private route: ActivatedRoute, private notificationService: NotificationService, private cdr: ChangeDetectorRef,
+    private router: Router, private spatialUtilService: SpatialUtilsService, private pointIdService: PointIdService,
+    private publishedIncidentService: PublishedIncidentService, private agolService: AGOLService) {
+
+  }
+
+  async ngOnInit(): Promise<void> {
+    this.route.queryParams.subscribe((params: ParamMap) => {
+      this.params = params
+    })
+    await this.fetchSavedLocation()
+    await this.fetchWeather()
+    await this.fetchFireBans()
+    this.fetchDangerRating()
+    await this.fetchEvacs()
+    await this.fetchNearbyWildfires()
+    this.getFireCentre(this.location)
+  }
+
+  async fetchSavedLocation() {
+    if (this.params && this.params['name']) {
+      await this.notificationService.getUserNotificationPreferences().then(response => {
+        if (response.notifications) {
+          for (const item of response.notifications) {
+            if (item?.notificationName === this.params['name'])
```

Comment:
I think notification name is not a primary key so just using the notification_name as param may be problematic. 
Could be more accurate to also use the lat and long as params? So use name + long + lat should match with one of the notifcations.

Reply:
Done, thank you